ui/components/ui/icon/approve-icon.component.js

```diff
@@ -1,6 +1,11 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
+/**
+ * @deprecated please use the <Icon /> component in ./ui/components/component-library/icon/icon.js
+ */
+
+
```

Comment:
```suggestion
```

Reply:
Hey @digiwand , 

I have made this change and added a new commit. Please review it.